openlcs/tasks/views.py

```diff
@@ -20,29 +70,51 @@ def list(self, request, *args, **kwargs):
             curl -X GET -H "Content-Type: application/json" \
 %(HOST_NAME)s/%(API_PATH)s/tasks/ -H 'Authorization: Token your_token'
 
+        ####__Supported query params__####
+
+        ``meta_id``, string, task meta id.
+
+        ``owner__username``, string, username of the task owner.
+
+        ``status``, string, Status of the task, possible values can be \
+``PENDING``, ``STARTED``, ``FAILURE``, ``SUCCESS``, ``RECEIVED``, ``REVOKED``, ``RETRY``.
+
+        ``params``, string, Params of the task, can use part of this field \
+to search.
+
+        ``date_done``, string, the day you want to filter the task, format: ``%%Y-%%m-%%d``.
+
+        ``traceback``, string, task traceback, can use part of this field to search
+
+
         ####__Response__####
 
             HTTP 200 OK
             Content-Type: application/json
 
             [
                 {
-                    "id": 1,
-                    "meta_id": "1e234842-0993-4e3a-8bb7-7dd9bed6f28c",
-                    "owner": "qduanmu",
-                    "params": "{\"package_nvr\": \"a2ps-4.14-23.el7\", \
-\"license_scan\": true, \"copyright_scan\": true}"
+                    "id": 20,
+                    "meta_id": "82b02650-5d36-4430-876a-c8401266b438",
+                    "owner": "admin",
+                    "params": "{\"package_nvr\": \"fio-3.1-2.el7\", \"license_scan\": true,
+                    \"copyright_scan\": true, \"product_release\": \"satellite-6.9.0\"}",
+                    "status": "SUCCESS",
+                    "date_done": "2022-10-24T09:17:07.406133",
+                    "traceback": null
                 },
                 {
-                    "id": 2,
-                    "meta_id": "c0e0fff0-1ab6-4f2b-851c-eaf969988df3",
-                    "owner": "qduanmu",
-                    "params": "{\"package_nvr\": \"ansible-2.4.2.0-2.el7\", \
-\"license_scan\": false, \"copyright_scan\": true, \"product_release\": \
-\"satellite-6.9.0\"}"
-                },
+                    "id": 19,
+                    "meta_id": "3808b332-45df-4fa2-a4ca-3225549adc2a",
+                    "owner": "admin",
+                    "params": "{\"package_nvr\": \"ansible-2.4.2.0-2.el7\", \"license_scan\": true,
+                     \"copyright_scan\": true, \"product_release\": \"satellite-6.9.0\"}",
+                    "status": "STARTED",
+                    "date_done": null,
+                    "traceback": null
+                }
             ]
-        """
+        """ # noqa
```

Comment:
If the code can move to newline, we don't need the `noqa`

Reply:
This is the doc text, i think i will more human readable to not follow the PEP8 line limit.Because it is no the code, i think it is ok to add a noqa tag.